queries/lua/injections.scm

```diff
@@ -190,3 +190,21 @@
 (comment
   content: (_) @injection.content
   (#set! injection.language "comment"))
+
+; vim.filetype.add({ pattern = { ["some lua pattern here"] = "filetype" } })
+((function_call
+  name: (_) @_filetypeadd_identifier
+  arguments:
```

Comment:
Is this field formatting intentional? @lucario387

Reply:
Yes

https://github.com/nvim-treesitter/nvim-treesitter/blob/93a9632b63c1a6878551b37688e0884c04178525/scripts/format-queries.lua#L204

Of course we can remove this to make it truly `:InspectTree`-like

Reply:
I think we should.